packages/styles/src/elements/reset.scss

```diff
@@ -142,3 +142,16 @@ th {
   border-style: solid;
   border-width: 0;
 }
+
+fieldset,
+legend {
+  padding: 0;
+}
+
+fieldset {
+  border: 0;
+}
+
+legend {
+  width: 100%;
+}
```

Comment:
This would belong into the element styles.

Reply:
The `width` style is moved to element styles.